splunk_connect_for_snmp_poller/utilities.py

```diff
@@ -141,3 +147,12 @@ def file_was_modified(file_path, last_mod_time):
         last_mod_time = os.stat(file_path, follow_symlinks=True).st_mtime
         return True, last_mod_time
     return False, last_mod_time
+
+
+def multi_key_lookup(dictionary, tuple_of_keys):
```

Comment:
can't find it used in the MR, if I missed it ignore this comment

Reply:
You're right. I have removed it.